static/js/popovers.js

```diff
@@ -488,6 +491,59 @@ exports.toggle_actions_popover = function (element, id) {
     }
 };
 
+exports.render_compose_actions_popover = function (element) {
+    var last_popover_elem = current_compose_actions_popover_elem;
+    popovers.hide_all();
+    if (last_popover_elem !== undefined
+        && last_popover_elem.get()[0] === element) {
+        // We want it to be the case that a user can dismiss a popover
+        // by clicking on the same element that caused the popover.
+        return;
+    }
+
+    var filter = narrow_state.filter();
+    var text_stream;
+    var text_conversation;
+    var stream_name;
+
+    if (!filter) {
+        text_stream = i18n.t("New stream message");
+        text_conversation = i18n.t("New private message");
+    } else {
+        if (filter.has_operator("is") && filter.operands("is")[0] === "private"
+            || filter.has_operator("pm-with") || filter.has_operator("group-pm-with")) {
+            text_stream = i18n.t("New stream message");
+            text_conversation = i18n.t("New conversation");
```

Comment:
Let's make this "New private message"

Reply:
FYI, this change means we don't alter the label of that button at all. So I just moved it to the template.